src/main/java/org/opennars/gui/package.java

```diff
@@ -1,3 +1,17 @@
+/**
```

Comment:
not wsure why but you have a duplicate header here. that doesnt usually happen...